api/src/pcapi/core/offers/repository.py

```diff
@@ -710,7 +712,10 @@ def get_filtered_stocks(
             sa.cast(models.Stock.beginningDatetime, sa.Time) >= time.replace(second=0),
             sa.cast(models.Stock.beginningDatetime, sa.Time) <= time.replace(second=59),
         )
-    return query.order_by(sa.desc(models.Stock.beginningDatetime)).limit(stocks_limit_per_page).all()
+    query = query.order_by(sa.asc(models.Stock.beginningDatetime))
```

Comment:
[détail] Pour un tri dans l'ordre ascendant, pas besoin de préciser `sa.asc()` : 

```suggestion
    query = query.order_by(models.Stock.beginningDatetime)
```